v2/brigadier/src/jobs.ts

```diff
@@ -1,15 +1,56 @@
 import { Event } from "./events"
+import { ConcurrentGroup, SerialGroup } from "./groups"
+import { Runnable } from "./runnables"
 
 const defaultTimeout: number = 1000 * 60 * 15
 
-export class Job {
+/**
+ * A Brigade job.
+ * 
+ * Instances of Job represent containers that your Brigade script can
+ * run using the Job#run method.
+ * 
+ * The Job#primaryContainer, initialised from the constructor image argument, determines
+ * how long the job runs and whether it is considered successful.By default, the container is simply
+ * executed (via its default entry point). Set the Job#primaryContainer#command
+ * property to run a specific command in the container instead.
+ * Other containers, specified via Job#sidecarContainers, are automatically
+ * terminated after the primary container completes.
+ * 
+ * Job also provides static methods for building up runnable
+ * elements out of more basic ones. For example, to compose
+ * a set of workloads into a sequence, you can use the
+ * Job.sequence method.
+ */
+export class Job implements Runnable {
+  /** The name of the jon. */
```

Comment:
s/jon/job

Reply:
Duh. I mean obviously the jon's name would be, er, Jon. Thanks.